APISpecification.yaml

```diff
@@ -128,6 +169,43 @@ paths:
 
 components:
   schemas:
+    BriefGenomeDetails:
+      type: object
+      properties:
+        genome_id:
+          type: string
+          example: a7335667-93e7-11ec-a39d-005056b38ce3
+        genome_tag:
+          type: string
+          example: grch38
+          nullable: true
+        common_name:
+          type: string
+          example: Human
+          nullable: true
+        scientific_name:
+          type: string
+          example: Homo sapiens
+        assembly_accession:
```

Comment:
This field should probably be named `assembly_accession_id` (see [spec for ensembl search](https://gitlab.ebi.ac.uk/ensembl-web/ensembl_search_hub/-/blob/master/APISpecification.yaml#L258))